webapp/channels/src/components/drafts/draft_row.tsx

```diff
@@ -129,7 +132,6 @@ function DraftRow({
         history.push(channelUrl);
     }, [channelUrl, dispatch, history, rootId, rootPostDeleted]);
 
-    // TODO LOL verify the types and handled it better
```

Comment:
This comment is still relevant, since we are still casting to any in `usePriority` and `useSubmit`.

Reply:
I've added the TODO again and will be dealing with it when implementing the "Send Now" functionality.